src/views/SignIn/index.js

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react';
+import { Alert, StatusBar } from 'react-native';
+import bgLogin from '../../assets/bg-login.png';
+import {
+    Title,
+    Subtitle,
+    Background,
+    FromContainer,
+    RegisterTitle,
+    RegisterButton,
+    ImageBackground,
+    RegisterContainer,
+    RegisterButtonText,
+} from './style';
+import { Button } from '../../components/Button';
+import { TextInputIcon } from '../../components/TextInputIcon';
+import { Colors } from '../../utils/Colors';
+
+export const SignIn = () => {
+    
+    const [user, setUser] = useState('');
+    const [password, setPassword] = useState('');
+    const [userView, setUserView] = useState(false);
+    const [errorUser, setErrorUser] = useState(false);
+    const [activeButton, setActiveButton] = useState(false);
+    const [passwordView, setPasswordView] = useState(false);
+    const [errorPassword, setErrorPassword] = useState(false);
+    const [activeRegisterButton, setActiveRegisterButton] = useState(false);
+
+
+    useEffect(() => {
+        setErrorUser(false)
+    },[user])
+
+    useEffect(() => {
+        setErrorPassword(false)
+    },[password])
+
+
+    const login = async() => {
+
+        if(!user || !password){
+            Alert.alert('Meus Beats', "Preencha o usuário e a senha")
+            return;
+        }
+
+        setActiveButton(!activeButton)
+
+        const raw = JSON.stringify({
+            "email": user,
+            "senha": password
+        });
+
+        const myHeaders = new Headers();
+        myHeaders.append("Content-Type", "application/json");
+    
+        var requestOptions = {
```

Comment:
Não é boa prática utilizar `var` no código
```suggestion
        const requestOptions = {
```